README.md

```diff
@@ -289,3 +289,68 @@ To classify documents, you must [create a custom document classifier in the Goog
   ```bash
   custom_classifier_id = projects/<CLASSIFIER_PROJECT>/locations/<CLASSIFIER_LOCATION>/processors/<CLASSIFIER_ID>
   ```
+
+### Connect to your AlloyDB instance
+
+The AlloyDB instance created as part of this solution, is deployed with no access outside the VPC. Furthermore, the data currently is saved within a new schema, `eks`, which is only available to the owning service account `eks-specialized-parser`.
+
+To view and interact with the data saved in AlloyDB, you must connect to your AlloyDB and grant permissions to another user that can be authenticated by you.
+
+One way to do this is to create a "Bastion" VM in your VPC, and use it to connect to your AlloyDB instance:
+
+1. Create a new VM inside your VPC:
+
+```bash
+gcloud compute instances create alloydb-bastion \
+    --project=<PROJECT_ID> \
+    --zone=<REGION>-a \
+    --machine-type=e2-medium \
+    --network-interface=network-tier=PREMIUM,stack-type=IPV4_ONLY,subnet=cloud-run-subnet \
```

Comment:
Flagging that refactoring where we create connectors and do the db config changed the context slightly...

subnet should now be "cloudrun-to-alloydb-connector-subnet"